plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -231,171 +286,121 @@ def read_next_set_of_high_ber_or_pdr_ports(self, endpoint_port):
             self.logger.error("Couldn't retrieve telemetry data")
             return issues
         for index, row in ports_counters.iterrows():
+            timestamp = row.get(Constants.TIMESTAMP) / 1000
```

Comment:
what will you get by / 1000?

Reply:
timestamp is received by telemetry in milisec so we convert it to seconds